.readthedocs.yml

```diff
@@ -1,16 +1,14 @@
 version: 2
 
-#conda:
-    #environment: docs/environment.yml
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
 
 python:
     install:
       - requirements: docs/rtd_requirements.txt
-      #- requirements: requirements.txt
       - requirements: docs/requirements.txt
       - method: setuptools
         path: .
-      #- method: pip
-        #path: .
-    #system_packages: false
-
+    system_packages: false
```

Comment:
isn't this the default anyway (and thus can be left out)? Assuming I read [this](https://docs.readthedocs.io/en/stable/config-file/v2.html#python-system-packages) correctly?
```suggestion
```

Reply:
For now, I want it there to ensure we're overriding anything set once-upon-a-time in the web interface. (The existence of the YAML file should override web settings, but this is just to be extra careful.) The specific event that triggered this PR was an email from RTD with the following paragraph:

> **We are removing the "use system packages" feature on August 29th.** Make sure you are installing all the required dependecies to build your project's documentation using a `requirements.txt` file and specifying it in your `.readthedocs.yaml`.

Reply:
seems reasonable to me